app/views/categories/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="container">
+  <div class="text-center">
+    <h1>Category</h1>
+  </div>
+  <div>
+    <ul class="list-group">
+      <% @categories.each do |cat| %>
+        <li class="list-group-item d-flex justify-content-between align-items-center w-100">
+          <%= link_to category_path(cat), class: "link link-underline-info" do %>
+            <div class="d-flex justify-content-between align-items-center me-3">
+              <div class="ms-2 me-auto d-flex align-items-center">
+                <img class="icon-images img-fluid" src="<%= cat.icon %>">
+                <div class="fw-bold px-2"><%= cat.name %></div>
+              </div>
+              <span class="badge bg-info rounded-pill link-underline-success">
+                <%= cat.total_amount %>
+              </span>
+            </div>
+          <% end %>
+          <div class="d-flex actions">
+            <%= link_to edit_category_path(cat) do %>
+              <button type="button" class="btn btn-outline-info me-2">Edit</button>
+            <% end %>
+            <%= button_to "Delete", cat, method: :delete, class: "btn btn-outline-danger" %>
+          </div>
+        </li>
+      <% end %>
+    </ul>
+  </div>
+  <div class="text-end my-5">
+    <button type="button" class="btn btn-outline-info"><%= link_to "New Category", new_category_path, class: "text-decoration-none text-dark"%></button>
+  </div>
+</div>
```

Comment:
- [ ] Kindly ensure your design matches the given design. Since we do not have a page for the categories page in the given design we should follow the transaction page which has been provided.
![image](https://github.com/SFarida/rails_capstone/assets/81932478/6e120a84-d3d7-48a8-987e-ce7a678f257f)

Your design
![image](https://github.com/SFarida/rails_capstone/assets/81932478/1c1c9d39-58af-4f15-bccc-1366d3b68684)

Given design
![image](https://github.com/SFarida/rails_capstone/assets/81932478/ec6c673d-301d-4763-b03e-b41ec349addb)

#### Note that as per the requirement, you should follow the given design including color, typography and layout of elements